src/code/codeResolver.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   },
 
   formatCode: function (hexCode) {
-    var code = codeUtils.nameOpCodes(new Buffer(hexCode.substring(2), 'hex'))
+    var code = codeUtils.nameOpCodes(Buffer.from(hexCode.substring(2), 'hex'))
```

Comment:
The only reason you could have seen a warning is `new Buffer` is deprecated in newer node. The problem with `Buffer.from` is that it was only introduced in node 5.1.